src/logger.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2017, Shopgate, Inc. All rights reserved.
+ *
+ * This source code is licensed under the Apache 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { green, blue } from 'chalk';
+
+/**
+ * The Logger class hold static functions for logging useful information when running
+ * the Rapid Dev Server and the Webpack Dev Server.
+ * @type {LogHelper}
+ */
+class LogHelper {
+  /**
+   * LogHelper.
+   */
+  constructor() {
+    /**
+     * A colored Shopgate Cloud prefix for console outputs.
+     * @type {string}
+     */
+    this.prefix = `${green('Shopgate')}${blue('Cloud')}`;
+  }
+
+  /**
+   * Logs a message
+   * @param {string} message The message
+   * @return {LogHelper}
+   */
+  log(message) {
+    /* eslint-disable no-console */
+    console.log(message);
```

Comment:
If you would define `const logger = console;` before the class definition or `this.logger = console;` in the constructor, you would not need to the disable and re-enable this eslint rule here. Therefore you could also test its calls.

Reply:
nice trick... did it